src/IndexBuilder.php

```diff
@@ -130,7 +130,7 @@ public function purgeOldIndices(string $indexName): array
 
         // Newest first
         uasort($indexes, function ($a, $b) {
-            return $a['date'] < $b['date'];
+            return $a['date'] < $b['date'] ? 1 : -1;
```

Comment:
Why not using the spaceship operator?
```suggestion
            return $b['date'] <=> $a['date'];
```

Reply:
Thanks, I always forget about this one :)